dbt/macros/check_row_counts_per_partition.sql

```diff
@@ -0,0 +1,18 @@
+{% test check_row_counts_per_partition(model, table_name, partition_column) %}
```

Comment:
Compares row counts in the table to expected counts which are stored in `seeds/row_counts.csv`